controllers/dataplane/openstackdataplanenodeset_controller.go

```diff
@@ -565,7 +565,8 @@ func checkDeployment(ctx context.Context, helper *helper.Helper,
 
 // SetupWithManager sets up the controller with the Manager.
 func (r *OpenStackDataPlaneNodeSetReconciler) SetupWithManager(
-	ctx context.Context, mgr ctrl.Manager) error {
```

Comment:
I don't know how I feel about these formatting changes. In principle they are fine, but it's kind of inflating the patch and polluting the blame. Not reason to object to the PR, obviously, but it would be better to avoid that.

Reply:
Yeah, true. I'll turn off auto formatting and restore these